dolphinscheduler-dao/src/main/resources/application-h2.yaml

```diff
@@ -18,7 +18,7 @@
 spring:
   datasource:
     driver-class-name: org.h2.Driver
-    url: jdbc:h2:mem:dolphinscheduler;MODE=MySQL;DB_CLOSE_DELAY=-1;DATABASE_TO_LOWER=true;INIT=runscript from 'classpath:sql/dolphinscheduler_h2.sql'
+    url: jdbc:h2:mem:dolphinscheduler;MODE=MySQL;DB_CLOSE_DELAY=-1;DATABASE_TO_LOWER=true
```

Comment:
You missed this part


https://github.com/apache/dolphinscheduler/blob/a00c4f9709a3235029a4911d79b0296fb8a3789d/dolphinscheduler-standalone-server/src/main/resources/application.yaml#L34-L36